plugin-rapl/Cargo.toml

```diff
@@ -6,7 +6,8 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-
+default = []
+test-support = ["tempfile", "nix"]
```

Comment:
these should be dev-dependencies, with the `test` cfg `#[cfg(test)]` instead of a custom feature